### DISCUSSION TITLE
Can't Access code-server, when deployed(EC2) Please Help!

### DISCUSSION THREAD

**Discussion by MC-Spraint** [coder/code-server#6757] [Q&A]
I've deployed code-server on my AWS EC2 instance, but I can't access it. All other servers are accessible on port 8080, when I run them but when I run code-server, I can't connect to it. I have tried different ports, nothing seems to solve the issue. Please help!!

**Comment by code-asher** [comment]
Copied from the [other discussion](https://github.com/coder/code-server/discussions/6756#discussioncomment-9104693):  

Just to be sure, did you also bind code-server to `0.0.0.0`?  If it is on `127.0.0.1` it will not be accessible outside the machine with the IP.
